### PR TITLE
Performance change in jaxrsmonitor

### PR DESCRIPTION
**Pull request by jdmcclur**
This change is to avoid a sync block in the jaxrs-monitor, it helps performance a small amount when mpMetrics is enabled under heavy load.

### CODE REVIEW

**Comment by jdmcclur** [comment]
#build

**Review by jhanders34 — APPROVED** [review]

File: dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/JaxRsMonitorFilter.java [hunk 1]
```diff
@@ -169,15 +169,20 @@ public void filter(ContainerRequestContext reqCtx, ContainerResponseContext resp
...
+								break;
 						}
+						minuteLatestMaximumDuration = stats.getMinuteLatestMaximumDuration();
+					}
 
-						if (elapsedTime < stats.getMinuteLatestMinimumDuration()
-						        || stats.getMinuteLatestMinimumDuration() == 0L) {
-							stats.updateMinuteLatestMinimumDuration(elapsedTime);
+					long minuteLatestMinimumDuration = stats.getMinuteLatestMinimumDuration();
+					while (elapsedTime < minuteLatestMinimumDuration || minuteLatestMinimumDuration == 0L) {
```
Comment (jhanders34) [inline dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/JaxRsMonitorFilter.java]: I kind of want to surround this while with:

```
if (!(elapsedTime == 0L && minuteLatestMinimumDuration == 0L)) {
```

Comment (jdmcclur) [reply dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/JaxRsMonitorFilter.java]: I added the if check.

**Comment by jdmcclur** [comment]
#build

**Comment by jdmcclur** [comment]
#libby

**Review by Channyboy — APPROVED** [review]
lgtm

**Pull request merged**